pkg/dd/variables.go

```diff
@@ -3,4 +3,7 @@ package dd
 import "time"
 
 // How long to cache repo list for
-var maxCacheAge = 180 * time.Minute
+var (
+	maxCacheAge = 180 * time.Minute
+	baseUrl     = "https://app.datadoghq.com"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [var baseUrl should be baseURL](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5619688)